tests/test_cli.py

```diff
@@ -396,10 +418,8 @@ def test_readme_command(self):
             self._main('buildreadme')
             # README and spec are opened
             self.assertEqual(mock_file.call_args_list[0][0],
```

Comment:
I'm starting to think we should just get rid of these assertions since the test results differ so much between jenkins and local testing.